packages/cli/src/lib/index.js

```diff
@@ -8,16 +8,17 @@ import decamelize from 'decamelize'
 import execa from 'execa'
 import Listr from 'listr'
 import VerboseRenderer from 'listr-verbose-renderer'
+import { memoize } from 'lodash'
 import lodash from 'lodash/string'
 import { paramCase } from 'param-case'
 import pascalcase from 'pascalcase'
 import { format } from 'prettier'
 
+import { getConfig as getRedwoodConfig } from '@redwoodjs/internal/dist/config'
 import {
   getPaths as getRedwoodPaths,
-  getConfig as getRedwoodConfig,
   resolveFile,
-} from '@redwoodjs/internal'
+} from '@redwoodjs/internal/dist/paths'
```

Comment:
Mind explaining this one? Are you trying to make it so that we just import path functions and not the kitchen sink with internal?

Reply:
@jtoar I think what we're trying to achieve is "tree shaking," but we're doing that manually here!

For a long running process I guess it's fine to pay the upfront cost of importing all this JavaScript, but for something like a CLI it's gonna hurt. I think we should use package.json "exports" to expose a light version of "@redwoodjs/internal" that we can use is middleware and for the `builder` parts of the CLI.